### PR TITLE
Migrate to new python plugin structure

### PR DESCRIPTION
**Pull request by Drimix20**
This PR resolve https://github.com/Drimix20/Flow.Launcher.Plugin.DirectTranslate/issues/1 New support for Python in Flow to use correct plugin structure defined by Flow Launcher. There is also fix issue https://github.com/Drimix20/Flow.Launcher.Plugin.DirectTranslate/issues/2 
List index out of range.

I also migrate to use googletrans for translation, because of textblob issue.

@deefrawley Can you please look at it and validate changes? I used https://github.com/Flow-Launcher/Flow.Launcher.Plugin.PythonTemplate template and I was inspired from your plugin.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+CONFIG=  # Here user can key in his own config.
+local=zh  # en or zh
```
Comment (deefrawley) [inline .env:2]: Just an FYI that this setting won't do anything as you don't have any translatable strings in your code. Shouldn't break anything and future proofs if you want to add translations later (might be worth doing given this is a translation plugin :P )

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+      - name: Set up Environment
+        run: |
+          python -m pip install --upgrade pip
+          python commands.py setup-env
+
+      - name: Install Dependencies to Local
+        run: python commands.py setup-env-to-lib
+
+      - name: Compile all languages
+        run: python commands.py compile
```
Comment (deefrawley) [inline .github/workflows/release.yml:30]: Again this won't update anything, and with the right folder structure in place it shouldn't affect this Action.

File: LICENSE [hunk 3]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2021 deefrawley
```
Comment (deefrawley) [inline LICENSE:3]: Replace my name with yours here

File: README.md [hunk 4]
```diff
@@ -1,17 +1,48 @@
-# Translate
-# Plugin for [Flow Launcher](https://flow-launcher.github.io/docs/#/) that translates between any languages supported by python textblob library.
+# Direct Translate (Flow.Launcher.Plugin.DirectTranslate)
+
+Translate plugin that translates between any languages supported by python textblob library for [Flow Launcher](https://github.com/Flow-Launcher/Flow.Launcher).
```
Comment (deefrawley) [inline README.md:3]: googletrans instead of textblob?

File: plugin/settings.py [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+# The default value can work, if no user config.
+CONFIG = os.getenv("CONFIG", "default config")
+LOCAL = os.getenv("local", "en")
+
+
+# the information of package
+__package_name__ = "Plugin"
+__version__ = "0.0.1"
+__short_description__ = "DESCRIPTION"
+GITHUB_USERNAME = "USERNAME"
```
Comment (deefrawley) [inline plugin/settings.py:27]: Update the above 4 lines with your plugin details

File: plugin/settings.py [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+PLUGIN_ID = "uuid"  # could generate via python `uuid` official package
+ICON_PATH = "assets/favicon.ico"
+PLUGIN_AUTHOR = "AUTHOR_NAME"
+PLUGIN_ACTION_KEYWORD = "KEYWORD"
+PLUGIN_PROGRAM_LANG = "python"
+PLUGIN_EXECUTE_FILENAME = "main.py"
+PLUGIN_ZIP_NAME = f"{__package_name__}-{__version__}.zip"
+PLUGIN_URL = f"https://github.com/{GITHUB_USERNAME}/{__package_name__}"
+PLUGIN_URL_SOURCE_CODE = f"https://github.com/{GITHUB_USERNAME}/{__package_name__}"
+PLUGIN_URL_DOWNLOAD = (
```
Comment (deefrawley) [inline plugin/settings.py:50]: And some more details here to add

File: plugin/utils.py [hunk 7]
```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
```
Comment (deefrawley) [inline plugin/utils.py:1]: If you don't use this file you can exclude it

**Review by deefrawley — CHANGES_REQUESTED** [review]

File: plugin/settings.py (hunk 6 above)
Comment (Drimix20) [reply plugin/settings.py:50]: Done.

File: .github/workflows/release.yml (hunk 2 above)
Comment (Drimix20) [reply .github/workflows/release.yml:30]: Translation addes, so I think it is used from now.

**Pull request merged**